packages/miniflare/turbo.json

```diff
@@ -3,10 +3,24 @@
 	"extends": ["//"],
 	"pipeline": {
 		"build": {
-			"outputs": ["dist/**"],
+			"inputs": [
+				"src/**",
+				"scripts/**",
+				"types/**",
+				"worker-metafiles/**",
+				"*.mjs",
+				"*.js",
+				"*.json"
+			],
+			"outputs": ["dist/**", "bootstrap.js"],
 			"env": [
 				"CI_OS"
 			]
+		},
+		"test": {
+			"inputs": [
+				"test/**"
```

Comment:
Doesn't this need to include `src` too?

Reply:
In the root turbo.json there is a relationship between test and build, which covers the src.
But I just realised that this should be "test:ci" and also it is probably better to make that dependency explicit here. So this should be more like:

```
		"test:ci": {
			"inputs": ["tests/**"],
			"dependsOn": ["build"]
		},
```